go.mod

```diff
@@ -1,12 +1,23 @@
 go 1.13
 
 require (
+	github.com/go-delve/delve v1.5.0 // indirect
+	github.com/go-logr/logr v0.1.0
```

Comment:
Do we need this dependency?

Reply:
The `logr` one gets added with `go mod tidy` so I think it's required.